### DISCUSSION TITLE
v2.4.3-Beta

### DISCUSSION THREAD

**Discussion by Gkiokan** [Gkiokan/ps4-remote-pkg-sender#10] [Show and tell]
Fixing the Storage creation error which causes the application crashes and endet up on the WhiteScreenOfDead.

The solution that I've implement may delay the start in case of an error within 1-2 secs because  
I am creating the store in a while Loop and catching any accuring error and force it to recreate the store.  

Added also a application global  errorHandler which should alert any other issue that accures.  

This should be ultimately kill any errors on windows ( which is the only Platform that had any problems yet ).  

Thanks for any feedback that helps me to fix the issues and make that tool better then ever.  

Spoiler: HB-Store implementation will be available in the next update v2.5.*

<hr /><em>This discussion was created from the release <a href='https://github.com/Gkiokan/ps4-remote-pkg-sender/releases/tag/v2.4.3'>v2.4.3-Beta</a>.</em>

**Comment by konansia** [comment]
Hello. When installing a game I get the error "Error 2157510677 | It seems to be installed already" as if the game is installed, but it really isn't.
I have searched for the game in storage, in Homebrew... but it is not there.
Any solution?
Thank you very much for your contributions
![Sin título](https://user-images.githubusercontent.com/71791288/151229000-29a28654-d892-43bf-b799-9ec08301ce77.png)

**Comment by Gkiokan** [comment]
Open your messages on ps4 and check if you have anything in downloading or smth. 
It can be installed or it is in the queue of the playstation that blocks the install process.  
Just delete it there and try again. Should fix it. If not restart your ps4 and try again.
